### DISCUSSION TITLE
How to run multiple flow

### DISCUSSION THREAD

**Discussion by Yuldoshev** [mobile-dev-inc/Maestro#860] [General]
Hello, how to run several flows at the same time?
for example : maestro test flow1.yaml flow2.yaml etc.

**Comment by axelniklasson** [comment]
Hi there @Yuldoshev! Please check the docs [here](https://maestro.mobile.dev/cli/test-suites-and-reports#running-multiple-tests) for information about how to run multiple tests. Thanks!
